plugins/connectrpc/dart/v0.1.0/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# syntax=docker/dockerfile:1.4
+FROM dart:3.5.4-sdk AS build
```

Comment:
What is the best way to know which dart version we should support?

Asking because there's an upstream issue where the directive is still at 2.x

https://github.com/google/protobuf.dart/issues/936

Reply:
Okay great, then we should be able to update the latest base types to the latest dart 3 version. That way we can keep both dart plugins on the same version.